codespell_lib/data/dictionary.txt

```diff
@@ -26129,6 +26132,8 @@ paermission->permission
 paermissions->permissions
 paeth->path
 pagagraph->paragraph
+pagent->pageant
```

Comment:
> `plangent` too?

Added.

> Perhaps even `agent`? Not sure about it, though it is certainly not a common typo: in GitHub stands for "permanent agent", "pAgent". I don't see anyone unwillingly typing `pagent` instead of `agent`.

Yup, I'd leave this one out.